ocrd/ocrd/processor/helpers.py

```diff
@@ -255,43 +255,47 @@ def wrap(s):
 
   %s%s
 
-Options:
+Options for processing:
+  -m, --mets URL-PATH             URL or file path of METS to process [./mets.xml]
+  -w, --working-dir PATH          Working directory of local workspace [dirname(URL-PATH)]
   -I, --input-file-grp USE        File group(s) used as input
   -O, --output-file-grp USE       File group(s) used as output
-  -g, --page-id ID                Physical page ID(s) to process
+  -g, --page-id ID                Physical page ID(s) to process [all]
```

Comment:
```suggestion
  -g, --page-id ID                Physical page ID(s) to process [default: all]
```

Though that might lead users to provide `--page-id all` explicitly...

Reply:
I thought of that interpretation, too. But I don't want to use `default` everywhere. Perhaps better like this:
```suggestion
  -g, --page-id ID                Subset of physical page ID(s) to process []
```
or
```suggestion
  -g, --page-id ID                Physical page ID(s) to process instead of full document []
```